### DISCUSSION TITLE
请问有做过子应用之间互相调用的吗？

### DISCUSSION THREAD

**Discussion by watermelonlth** [umijs/qiankun#2570] [Q&A]
应用A调用应用B的某一个页面

**Comment by ebdmn** [comment]
你好，你这个实现了吗？我也正好需要这个需求

**Comment by humorHan** [comment]
> 你好，你这个实现了吗？我也正好需要这个需求

可以具体下诉求，这描述太抽象了...

**Reply by ebdmn** [reply]
我想在一个子应用里加载另一个子应用的 vue 页面（两个子应用技术栈相同）

**Comment by Zuojiangtao** [comment]
`在一个子应用里加载另一个子应用的 vue 页面` 这个嵌场景套，你可以看下：https://github.com/umijs/qiankun/issues/1777#issuecomment-954336562 。
